src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label
+      for="search-big"
+      class="search-big"
```

Comment:
Please re-read checklist again and provide 1 general class for inputs and 1 additional class for big/small search
![image](https://user-images.githubusercontent.com/54936044/225774919-28b4683b-34f3-4f66-847d-c8b2420d64c7.png)